app/assess/routes.py

```diff
@@ -219,6 +224,38 @@ def flag(application_id):
     )
 
 
+@assess_bp.route("/qa_complete/<application_id>", methods=["GET", "POST"])
+@login_required(roles_required=["LEAD_ASSESSOR"])
+def qa_complete(application_id):
+
+    form = MarkQaCompleteForm()
+
+    if request.method == "POST" and form.validate_on_submit():
```

Comment:
`form.validate_on_submit()` already checks for `POST` method: https://github.com/wtforms/flask-wtf/blob/1cf8b19bcbc53e5357a4aa0c5ccb5cf7d3ea0d22/src/flask_wtf/form.py#L127

Reply:
done!